Rmd/DESeq2_report_new.Rmd

```diff
@@ -159,26 +159,28 @@ bscols(widths = c(3,NA),
 
 ```
 
-# PCA plots {.tabset .tabset-fade}
+# PCA plots
 
 ```{r PCA_function, collapse=TRUE}
 
-plot_interactive_PCA <- function(rld_data){
+plot_interactive_PCA <- function(rld_data, intgroup_to_plot){
   ## Perform PCA analysis
-  data_pca <- plotPCA(rld_data, intgroup = params$design, ntop = nrow(assay(rld_data)), returnData = TRUE)
+  data_pca <- plotPCA(rld_data, intgroup = c("diet_dose","Diet","dose","RNA_extraction"), ntop = nrow(assay(rld_data)), returnData = TRUE)
```

Comment:
Looks like you've unintentionally hard coded some variables here

Reply:
yeah that's why it's unfinished :D I'm still testing